src/container/srv_container.c

```diff
@@ -5273,6 +5311,12 @@ cont_op_save_dup(struct rdb_tx *tx, struct ds_pool_hdl *pool_hdl, struct cont_sv
 	crt_opcode_t              opc = opc_get(rpc->cr_opc);
 	int                       rc  = 0;
 
+	/* Test here for simpler caller invocation code */
+	if (dup_op)
+		goto out;
```

Comment:
[Nit; not requesting changes] This reminds me that the `cont_op_save_dup` name sounds like "save duplicate" and this "if" statement seems to contradict the name, whereas what we really want is
```
if not a duplicate
    save result
```
I hope I didn't misunderstand this function.

Reply:
The revised code looks much easier to read, I think. Thanks.